With two lychee-action steps in a row you get -> tar: lychee: Cannot open: File exists

### Intro

I have a huge list of files to analyze and I decided to split `URL` and `email` validation to different steps

```
...
    - name: Run lychee for emails
      uses: lycheeverse/lychee-action@v1.6.1
      with:
        args: --no-progress --exclude "(https?)|(file)://" --cache --max-cache-age 1d '_site/**/*.html'
        fail: true

    - name: Run lychee for links
       with:
        args: --no-progress --exclude-mail --cache --max-cache-age 1d '_site/**/*.html'
        fail: true
...
```

### Problem

In the second step, I get:

```
...
lychee
tar: lychee: Cannot open: File exists
tar: Exiting with failure status due to previous errors
Error: Process completed with exit code 2.
...
```

Probably it happens because `tar` has already been downloaded.
Probably the right solution is to reuse existing tag/extracted files it they exists

### Workaround

```
...
    - name: Run lychee for emails
      uses: lycheeverse/lychee-action@v1.6.1
      with:
        args: --no-progress --exclude "(https?)|(file)://" --cache --max-cache-age 1d '_site/**/*.html'
        fail: false

    - name: Run lychee for links
      run: lychee --no-progress --exclude-mail --cache --max-cache-age 1d '_site/**/*.html'
...
```

--- a/action.yml
+++ b/action.yml
@@ -38,11 +38,12 @@ runs:
   steps:
     - name: Install lychee
       run: |
-        curl -sLO 'https://github.com/lycheeverse/lychee/releases/download/v${{ inputs.LYCHEEVERSION }}/lychee-v${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz'
-        tar -xvzf lychee-v${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz
-        install -t $HOME/.local/bin -D lychee
+        curl -sLO "https://github.com/lycheeverse/lychee/releases/download/v${{ inputs.LYCHEEVERSION }}/lychee-v${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
+        tar -xvzf "lychee-v${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
+        rm "lychee-v${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
+        install -D lychee "$HOME/.local/bin" 
         rm lychee
-        echo "$HOME/.local/bin" >> $GITHUB_PATH
+        echo "$HOME/.local/bin" >> "$GITHUB_PATH"
       shell: bash
     - name: Run lychee
       run: ${{ github.action_path }}/entrypoint.sh

--- a/action.yml
+++ b/action.yml
@@ -41,7 +41,7 @@ runs:
         curl -sLO "https://github.com/lycheeverse/lychee/releases/download/v${{ inputs.LYCHEEVERSION }}/lychee-v${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
         tar -xvzf "lychee-v${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
         rm "lychee-v${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
-        install -D lychee "$HOME/.local/bin" 
+        install -t "$HOME/.local/bin" -D lychee 
         rm lychee
         echo "$HOME/.local/bin" >> "$GITHUB_PATH"
       shell: bash

